lk2nd/device/dts/msm8909/msm8909-qrd-skub.dts

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8909 0>;
+	qcom,board-id = <QCOM_BOARD_ID_QRD 0x09>,
+			<QCOM_BOARD_ID(QRD, 1, 0) 0x09>;
+};
+
+&lk2nd {
+	// The motherboard of it has QR code sticker has a code name printed on it.All Lenovo Yoga Tab 3 10 LTE/WiFi varian code name appear to have lxf-p5100 prefix and a lot of different suffix. this device is particular known for having b1 suffix and yt3-x50y model number.
```

Comment:
I'd appreciate if this comment were to be C style (`/**/`) and folded at 60-80. This would make it easier to read